cmd/kube-proxy/app/server_others.go

```diff
@@ -367,7 +338,7 @@ func newProxyServer(
 		OOMScoreAdj:            config.OOMScoreAdj,
 		ConfigSyncPeriod:       config.ConfigSyncPeriod.Duration,
 		HealthzServer:          healthzServer,
-		UseEndpointSlices:      useEndpointSlices,
+		UseEndpointSlices:      true, // todo (knabben) change all places
```

Comment:
`TODO` or `FIXME` is more standard

Reply:
My intent is to remove the options and keep the true code path, going to fix the todo yet on this PR.